lib/src/builders/interaction_response.dart

```diff
@@ -128,6 +128,8 @@ class ModalBuilder extends CreateBuilder<ModalBuilder> {
   Map<String, Object?> build() => {
         'custom_id': customId,
         'title': title,
-        'components': components.map((e) => e.build()).toList(),
+        'components': [
+          for (TextInputBuilder component in components) ActionRowBuilder(components: [component]).build()
+        ],
```

Comment:
Could you change `components` from `List<TextInputBuilder>` to a `List<ActionRowBuilder>`, and then:
```suggestion
        'components': components.map((e) => e.build()).toList(),
```
This would avoid us having to make a breaking change if Discord ever adds support for different components in modals.

Reply:
Wouldn't that break `nyxx-commands` as it is expecting to send `List<TextInputBuilder>` to `ModalBuilder`?

Reply:
nyxx_commands can always be updated

Reply:
PR is targeting 6.0 so we can intruduce breaking changes